zulipterminal/ui_tools/views.py

```diff
@@ -925,16 +933,16 @@ def keypress(self, size: urwid_Size, key: str) -> str:
 
 
 class StreamInfoView(PopUpView):
-    def __init__(self, controller: Any, color: str,
-                 name: str, desc: str) -> None:
-        # Add 4 (for 2 Unicode characters on either side) to the popup title
-        # length to make sure that the title gets displayed even when the
-        # content is shorter than the title length (+4 Unicode characters).
-        self.width = max(len(desc) + 2, len("# {}".format(name)) + 4)
-        description = urwid.Text(desc, align='center')
```

Comment:
While centered and padded content will look good as a standalone item, it will a bit odd in popups which also have two columns (where the left column is left and the right column is right aligned).

Re 'snug', I agree. I see that you've suggested 'mimicking the existing code for now' (which I assume means using the left column for text and leaving the right column blank). 

On another note, the `StreamInfoView` looks good without the changes that are being introduced in this commit. We could also hold off this commit as "we don't know what we'll need in the future".

Thoughts?

Reply:
I was meaning to use padded and centered style on the single-column version, ie. the specialized version introduced in that commit, to retain the existing style. So two columns => help menu style, text => StreamInfo style (as it is now).

We could leave this commit for a later merge in this PR or elsewhere. Once things are tidied from the other comments I could merge some commits and we could either work on the remaining commit(s) here or move them to another branch/PR.

Reply:
I understand what you're suggesting. Though, I am concerned about the scenario where we would want to have both in the same popup i.e. the help menu style and the stream info style. It would look a bit off to have some content centre-aligned and some left/right-aligned. That said, if we decide to stick with either one of the styles in any popup (not mix single-column and two-column), that would work.